src/libraries/Common/src/Interop/OSX/System.Security.Cryptography.Native.Apple/Interop.Aead.cs

```diff
@@ -29,18 +31,17 @@ internal static partial class AppleCrypto
             fixed (byte* tagPtr = tag)
             fixed (byte* aadPtr = aad)
             {
-                const int Success = 1;
-                int result = AppleCryptoNative_ChaCha20Poly1305Encrypt(
-                    keyPtr, key.Length,
-                    noncePtr, nonce.Length,
-                    plaintextPtr, plaintext.Length,
-                    ciphertextPtr, ciphertext.Length,
-                    tagPtr, tag.Length,
-                    aadPtr, aad.Length);
-
-                if (result != Success)
+                AppleCryptoNative_ChaCha20Poly1305Encrypt(
+                    new(keyPtr, key.Length),
```

Comment:
Generally we would not use a target type new here. TTN should be used only when the type name is in a variable declaration on the LHS.